analysis/diabetes_analysis.ipynb

```diff
@@ -25,6 +41,32 @@
     "from sklearn.linear_model import LogisticRegression"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "0f464a8d-35d0-4b26-8013-3f6481017712",
+   "metadata": {},
+   "source": [
+    "# Summary"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8377f0b6-b44c-4a63-94aa-d4f7ed1496dc",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "Diabetes is a serious chronic disease characterized by high levels of glucose in the blood, caused by either insufficient insulin production by the pancreas or the body’s inability to effectively use insulin. It has become a significant global health issue, with its prevalence nearly doubling since 1980, rising from 4.7% to 8.5% in adults by 2014 (World Health Organization, 2016). Early detection enables timely interventions, reduces complications, lowers healthcare costs, and improves quality of life and long-term outcomes (Marshall & Flyvbjerg, 2006).\n",
+    "\n",
+    "Artificial intelligence (AI) leverages computer systems and big data to simulate intelligent behavior with minimal human intervention, and within it, machine learning (ML) is a subset of AI methodologies. Machine learning has increasingly been applied in various areas of disease detection and prevention in the healthcare field (Bini, 2018). Numerous machine learning techniques have been deployed to develop more efficient and effective methods for diagnosing chronic diseases (Battineni, Chinatalapudi, & Amenta, 2020). Utilizing machine learning methods in diabetes research is a critical strategy for harnessing large volumes of diabetes-related data to extract valuable insights (Agarwal & Vadiwala, 2022).\n",
+    "\n",
+    "The goal of this report is to leverage a supervised machine learning model, logistic regression (LR), to evaluate its predictive performance in diagnosing diabetes using a real-world dataset focused specifically on Pima Indian women aged 21 and older. The dataset used in this study originates from the National Institute of Diabetes and Digestive and Kidney Diseases and we sourced from the UCI Machine Learning Repository (Dua & Graff, 2017). The dataset can be found here. The primary objective of the dataset is to enable diagnostic prediction of whether a patient has diabetes based on specific diagnostic measurements.\n",
+    "\n",
+    "To ensure consistency and relevance, several constraints were applied to the selection of data instances. Specifically, the dataset includes only female patients who are at least 21 years old and of Pima Indian heritage. Each record in the dataset represents a patient and contains measurements of key health indicators, such as glucose levels, blood pressure, and body mass index (BMI). Additionally, predictor variables such as the number of pregnancies and age are included, offering comprehensive features for machine learning analysis.\n"
```

Comment:
Same comment as above. 

Please try to merge with Inder's Data section. Remove any repeats / redunduncies while keeping valuable info that serve as good addition to his write up.